packages/core/admin/admin/src/content-manager/pages/ListView/components/BulkActionButtons/index.js

```diff
@@ -69,8 +71,44 @@ const confirmDialogsPropTypes = {
  * ConfirmDialogPublishAll
  * -----------------------------------------------------------------------------------------------*/
 
-const ConfirmDialogPublishAll = ({ isOpen, onToggleDialog, isConfirmButtonLoading, onConfirm }) => {
+const ConfirmDialogPublishAll = ({
+  isOpen,
+  onToggleDialog,
+  isConfirmButtonLoading,
+  onConfirm,
+  slug,
+}) => {
   const { formatMessage } = useIntl();
+  const { get } = useFetchClient();
+  const { selectedEntries } = useTableContext();
+
+  const { data: countDraftRelations } = useQuery(
```

Comment:
do we not want error handling here e.g. what does the user do if the network call fails?

Reply:
Yes, we decided to handle the errors in the same way we do when we publish a single Entry from its Edit view (we don't show the notification message and instead we show an error)